packages/shared/lib/services/sync/orchestrator.service.ts

```diff
@@ -244,23 +274,44 @@ export class Orchestrator {
             }
 
             for (const sync of syncs) {
-                const schedule = await getSchedule(sync?.id as string);
-                const syncClient = await SyncClient.getInstance();
-                await syncClient?.runSyncCommand(schedule?.schedule_id as string, sync?.id as string, command, activityLogId as number, environmentId);
-                if (command !== SyncCommand.RUN) {
-                    await updateScheduleStatus(schedule?.schedule_id as string, command, activityLogId as number, environmentId);
+                const schedule = await getSchedule(sync.id as string);
+                if (!schedule) {
+                    continue;
+                }
+
+                if (fullResync) {
```

Comment:
Probably out of scope but I don't like how this logic is repeated (probably my fault 🤦🏿‍♂️ from earlier implementation)

Reply:
The method takes an optional `connectionId` it creates two paths in this method